pkg/app/app_list_test.go

```diff
@@ -242,7 +242,7 @@ environments:
 releases:
 - name: myrelease1
   chart: mychart1
-  installed: no
```

Comment:
It turned out on/off and yes/no are deprecated in YAML 1.2.
https://github.com/goccy/go-yaml/issues/214

To make it backward-compatible for Helmfile v.0x, I'd like to revert Helmfile v0.x to use `gopkg.in/yaml.v2` by default.
Let feature the deprecation in the helmfile v1 proposal and enable goccy/go-yaml only for Helmfile v1, so that we won't break anything for Helmfile v0.x while we can still migrate to a maintained yaml library.

Reply:
ok